src/Mailer/Email.php

```diff
@@ -1910,9 +1909,7 @@ protected function _attachFiles($boundary = null)
      */
     protected function _readFile($path)
     {
-        $File = new File($path);
-
-        return chunk_split(base64_encode($File->read()));
+        return chunk_split(base64_encode((string)file_get_contents($path)));
```

Comment:
This introduces a security risk. Now it is possible to fetch remote resources thanks to `file_get_contents`. We need to make sure that the file exists first.

Reply:
Should an exception be thrown is `file_exists()` check fails?

Reply:
Actually no change is needed. `Email::setAttachments()` already uses `realpath()` and `file_exists()` check and throws exception.